apps/bare-expo/android/build.gradle

```
@@ -4,13 +4,13 @@ import groovy.json.JsonSlurper
 
 buildscript {
     ext {
-        buildToolsVersion = "28.0.3"
+        buildToolsVersion = "29.0.3"
```

Are you sure there is such version available? Accordint to [this page](https://developer.android.com/studio/releases/build-tools) `29.0.2` is newest available, and we should go with it.
```suggestion
        buildToolsVersion = "29.0.2"
```